natvis/cppwinrt.natvis

```diff
@@ -20,12 +20,15 @@
     <DisplayString Condition="this == 0">null</DisplayString>
   </Type>
   <!--Primitive type visualizers-->
+  <Type Name="winrt::Windows::UI::Color">
+    <DisplayString># {A,nvoXb} {R,nvoXb} {G,nvoXb} {B,nvoXb}</DisplayString>
```

Comment:
Please remove the spaces and confirm that leading zeroes are emitted.

Reply:
I wish there was a "half-space" character. I added the spaces so it would be easier to see each byte. I will confirm the single-digit case.

Reply:
I've removed the spaces and confirmed that, yes, byte values less than 0x10 have the leading zero added.